tests/query/atomWithQuery.test.tsx

```diff
@@ -301,5 +302,7 @@ it('query with initialData test', async () => {
 
   // NOTE: the atom never suspends
   await findByText('count: 0')
+  expect(mockFetch).toHaveBeenCalledTimes(0)
```

Comment:
@dai-shi does this work for you with regards to testing? I can't think immediately a way to create a failing test

Reply:
Do you mean this doesn't fail before the fix?

Reply:
ah i see what you mean, yes it does fail:

![Screen Shot 2021-07-09 at 6 41 09 PM](https://user-images.githubusercontent.com/11803153/125144970-4a15b000-e0e5-11eb-83c3-10113887e83e.png)